pvlib/iotools/tmy.py

```diff
@@ -519,3 +519,179 @@ def _read_tmy2(string, columns, hdr_columns, fname):
+    year
+    month
+    day
+    hour
+    minute
+    data_source_unct                    Data source and uncertainty flags. See [1], chapter 2.13
+    t_drybulb                           Dry bulb temperature at the time indicated, deg C
+    t_dewpoint                          Dew-point temperature at the time indicated, deg C
+    rel_hum                             Relatitudeive humidity at the time indicated, percent
+    atmospheric_pressure                Station pressure at the time indicated, Pa
+    etr                                 Extraterrestrial horizontal radiation recv'd during 60 minutes prior to timestamp, Wh/m^2
+    etrn                                Extraterrestrial normal radiation recv'd during 60 minutes prior to timestamp, Wh/m^2
+    ghi_infrared                        Horizontal infrared radiation recv'd during 60 minutes prior to timestamp, Wh/m^2
+    ghi                                 Direct and diffuse horizontal radiation recv'd during 60 minutes prior to timestamp, Wh/m^2
+    dni                                 Amount of direct normal radiation (modeled) recv'd during 60 mintues prior to timestamp, Wh/m^2
+    dhi                                 Amount of diffuse horizontal radiation recv'd during 60 minutes prior to timestamp, Wh/m^2
+    global_hor_illum                    Avg. total horizontal illuminance recv'd during the 60 minutes prior to timestamp, lx
+    direct_normal_illum                 Avg. direct normal illuminance recv'd during the 60 minutes prior to timestamp, lx
+    diffuse_horizontal_illum            Avg. horizontal diffuse illuminance recv'd during the 60 minutes prior to timestamp, lx
+    zenith_luminance                    Avg. luminance at the sky's zenith during the 60 minutes prior to timestamp, cd/m^2
+    wind_direction                      Wind direction at time indicated, degrees from north (360 = north; 0 = undefined,calm)
+    wind_speed                          Wind speed at the time indicated, meter/second
+    total_sky_cover                     Amount of sky dome covered by clouds or obscuring phenonema at time stamp, tenths of sky
+    opaque_sky_cover                    Amount of sky dome covered by clouds or obscuring phenonema that prevent observing the sky at time stamp, tenths of sky
+    visibility                          Horizontal visibility at the time indicated, km
+    ceiling_height                      Height of cloud base above local terrain (7777=unlimited), meter
+    present_weather_observation         Indicator for remaining fields: If 0, then the observed weather codes are taken from the following field. If 9, then “missing” weather is assumed. Since the primary use of these fields (Present Weather Observation and Present Weather Codes) is for rain/wet surfaces, a missing observation field or a missing weather code implies no rain.
```

Comment:
the quotes around missing on this line are not standard ascii. probably caused by copy/paste issues. likely can be fixed by deleting them and retyping them, though I don't know that they're actually necessary.

Reply:
fixed in new commit